app/assets/stylesheets/cc-runtime-base.scss

```diff
@@ -174,11 +174,6 @@ div.activity-mod>div.site-width {
   }
 }
 
-// *********************************************
-//  7. Sticky Interactives
-// *********************************************
-@import "partials/runtime-sticky";
-
 // *********************************************
 //  8. Interactive + Questions
 // *********************************************
```

Comment:
Probably ought to update the number of the sections.

Reply:
Done.